test/test.py

```diff
@@ -356,11 +357,30 @@ def test_file_core_bundle(self):
                                    library_construction_methods={"10x 3' v3 sequencing"},
                                    content_description={'DNA sequence'})
         submitter_id_counts = Counter(file.submitter_id for file in bundle.files.values())
-        expected_counts = {
+        self.assertEqual(submitter_id_counts, {
```

Comment:
Expected should be first argument. And why do you inline one argument but extract the other into a variable? Both are relatively long expressions.